templates/SilverStripe/Blog/Includes/BlogSideBar.ss

```diff
@@ -0,0 +1,23 @@
+<nav role="navigation" class="page-sidebar-widget page-sidebar-nav">
+  <% if $BlogPage.Categories %>
+      <h3 class="h4 page-sidebar-header"><%t Bambusa.CATEGORIES "Browse by category" %></h3>
```

Comment:
Any chance we could move that translation key to the blog module?

Reply:
The blog module does not natively support "browse by category," so it would be an unused translation and a mix of concerns. That functionality is supposed to be handled by widgets/content blocks.